models/OnboardClusterConfig.go

```diff
@@ -0,0 +1,49 @@
+package models
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+)
+
+type OnboardNamespace struct {
+	Enablemonitoring bool
+	Name             string
+	Quota            string
+}
+
+type OnboardingRequest struct {
+	Env                string `yaml:",omitempty"`
+	Namespaces         []OnboardNamespace
+	ProjectDescription string `yaml:"project_description"`
+	Targetcluster      string `yaml:"target_cluster"`
+	Teamname           string `yaml:"team_name"`
+	Template           string
+	Users              []string `yaml:",omitempty"`
+}
```

Comment:
We need a section of the documentation that describes the input data expected by the `onboard` command (and shows a complete example).